action.yml

```diff
@@ -121,6 +125,21 @@ runs:
             TWINE_OPTS="$TWINE_OPTS --skip-existing"
           fi
           python -m pip install twine
+          if [[ -z "$TWINE_PASSWORD" ]]; then
+            # vis https://docs.pypi.org/trusted-publishers/using-a-publisher/#the-manual-way
+            export TWINE_PASSWORD=$(
+        python -c 'import json, urllib.request as req
+        audience = req.urlopen("'${INPUT_REGISTRY_DOMAIN}'/_/oidc/audience").read()
+        token = json.load(req.urlopen(req.Request(
+            url="'${ACTIONS_ID_TOKEN_REQUEST_URL}'/_/oidc/audience",
+            data=audience,
+            headers={"Authorization": "bearer '${ACTIONS_ID_TOKEN_REQUEST_TOKEN}'"})))
+        print(json.load(req.urlopen(req.Request(
+            url="https://'${INPUT_REGISTRY_DOMAIN}'/_/oidc/github/mint-token",
+            data=json.dumps({"token": token["value"]}).encode("U8"),
```

Comment:
Consider calling `.encode()` sans any parameters; default is UTF-8.

Reply:
```suggestion
            data=json.dumps({"token": token["value"]}).encode(),
```